assets/js/base/components/cart-checkout/product-sale-badge/index.js

```diff
@@ -25,10 +26,16 @@ const ProductSaleBadge = ( { currency, saleAmount } ) => {
 	}
 	return (
 		<div className="wc-block-sale-badge">
-			{ sprintf(
-				/* translators: %s discount amount */
-				__( 'Save %s!', 'woo-gutenberg-products-block' ),
-				formatPrice( saleAmount, currency )
+			{ __experimentalCreateInterpolateElement(
+				__( 'Save <price></price>!', 'woo-gutenberg-products-block' ),
```

Comment:
You should be able to just do a self-closing `<price />` here.  Also might be good to add a translators comment just to make it clear that `<price/>` should not be translated.